test/hotspot/gtest/opto/test_constant_division.cpp

```diff
@@ -23,14 +23,16 @@
  */
 
 #include "precompiled.hpp"
+#include "opto/divconstants.hpp"
 #include "runtime/os.hpp"
-#include "utilities/globalDefinitions.hpp"
 #include "utilities/growableArray.hpp"
 #include <type_traits>
 #include "unittest.hpp"
 
+// Generate a random positive integer of type T in a way that biases
+// towards smaller values
```

Comment:
Why is there a bias toward smaller numbers?  Maybe it should be named differently to indicate that bias?

Reply:
Because we are dealing with inputs of division so it makes more sense to have them following somewhat a reciprocal distribution.